core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsBillingProjectComponent.scala

```diff
@@ -285,6 +285,15 @@ trait RawlsBillingProjectComponent {
         .withProjectName(projectName)
         .setCreationStatus(status, message)
 
+    /**
+      * Whenever this is called, the status is implied to be CreationStatus.Ready
```

Comment:
🤔 I don't follow this comment given the code below -- how does this method affect the billing project status?

Reply:
I just forgot to remove that comment after making some changes. I'll remove it.